upstash_redis/client.py

```diff
@@ -44,6 +45,7 @@ def __init__(
         :param rest_retries: how many times an HTTP request will be retried if it fails
         :param rest_retry_interval: how many seconds will be waited between each retry
         :param allow_telemetry: whether anonymous telemetry can be collected
+        :param read_your_writes: whether the client should wait for the response of a write operation before sending the next one
```

Comment:
Not exactly a wait... Writing the same comment from JS SDK PR:
```
When this flag is enabled, any subsequent commands issued by this client are guaranteed to observe the effects of all earlier writes submitted by the same client.
```

Reply:
Updated the docstring